api/src/main/java/io/strimzi/api/kafka/model/KafkaConnect.java

```diff
@@ -40,7 +39,27 @@
                 ),
                 group = KafkaConnect.RESOURCE_GROUP,
                 scope = KafkaConnect.SCOPE,
-                version = KafkaConnect.VERSION
+                version = KafkaConnect.V1BETA1,
+                versions = {
+                        @Crd.Spec.Version(
+                                name = KafkaConnect.V1BETA1,
+                                served = true,
+                                storage = true
+                        ),
+                        @Crd.Spec.Version(
+                                name = KafkaConnect.V1ALPHA1,
+                                served = true,
+                                storage = false
+                        )
+                },
+                additionalPrinterColumns = {
+                        @Crd.Spec.AdditionalPrinterColumn(
+                                name = "Replicas",
```

Comment:
See comment above about `Kafka` resource. You used "scale" instead of "replicas" there.

Reply:
What do we think is the right terminology? Kube uses "Desired" (and "Current") in its output for `Deployment` and `StatefulSet`, rather than "replicas". But the picture is more complicated for us where we have Kafka and Zookeeper (each with 'desired' and 'current', even if we can't support 'current' until we have status subresources) for `Kafka` resources, and 'partitions' and 'replicas' for `KafkaTopic`s.

Reply:
I agree on using "Replicas" here. I would change "Kafka scale" in "Kafka replicas" as my comment above. Even using "Zookeeper replicas" maybe.

Reply:
Really it should be "desired Kafka replicas" and "desired Zookeeper replicas". Do we have the ambition to also export the current scale of these things via `status`? (Keeping them properly up to date would require a watch on the STS, so I'm inclined not to try doing this, since it's only of marginal value). But in any case the fact that it's the _desired_ scale is important.

Reply:
Good point, but in any case "Kafka scale" doesn't give me the idea on what it means. Then "desired XXXX replicas" seems to be too long. I am running out of good short names :-(

Reply:
I've added desired everywhere. We can always change it later.